packages/api/endpoints/granules.js

```diff
@@ -471,6 +471,7 @@ async function bulkOperations(req, res) {
       payload,
       type: 'BULK_GRANULE',
       envVars: {
+        ES_HOST: process.env.ES_HOST,
```

Comment:
This environment variable is needed for Elasticsearch logic. Otherwise, I was seeing 
```
  "errorType": "ConnectionError",
  "errorMessage": "connect ECONNREFUSED 127.0.0.1:9200",
```

Reply:
I thought it was out of the scope of the ticket to investigate why the `esHost` variable is passed into `startAsyncOperation` but seems unused https://github.com/nasa/cumulus/pull/2593/files#diff-624123a7dc5bb5513e3769b27fe85821577087609a9376829676b7edbeb3a5aeL484

Reply:
It's a bit tricky but you do need these because the `bulkOperation` Lambda uses any values passed in from `payload.envVars` to see environment variables:

https://github.com/nasa/cumulus/blob/feature/rds-phase-2/packages/api/lambdas/bulk-operation.js#L199